### PR TITLE
Updates to Shor's algorithm notebook

### PR DESCRIPTION
**Pull request by mmvandieren**
### Description

removed all endian references.  Double checked all the circuits so that the qubit order is correct for the operation intended.

### CODE REVIEW

File: docs/sphinx/examples/python/tutorials/Shors.ipynb [hunk 1]
```diff
@@ -282,7 +302,7 @@
     "# Edit the value in the line below to try out different initial guesses for a.\n",
     "# What happens when you choose a = 42 for the integer 123?\n",
     "# What happens when you choose a = 100 for the integer 123?\n",
-    "initial_value_to_start = 42  #edit this value\n",
+    "initial_value_to_start = 42  #edit this value it should be less than my_integer\n",
```
Comment (bmhowe23) [inline docs/sphinx/examples/python/tutorials/Shors.ipynb:305]:
```suggestion
    "initial_value_to_start = 42  # edit this value; it should be less than my_integer\n",
```

File: docs/sphinx/examples/python/tutorials/Shors.ipynb [hunk 2]
```diff
@@ -21,15 +21,35 @@
...
+   "id": "486fb3d4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: contfrac in /usr/local/lib/python3.10/dist-packages (1.0.0)\n",
+      "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
+      "\u001b[0m"
```
Comment (bmhowe23) [inline docs/sphinx/examples/python/tutorials/Shors.ipynb:42]: I would recommend clearing this output cell in order to clean up the web page that will be generated from this notebook. (No need for regular web page viewers to see these warnings about broken things 😄 ).

File: docs/sphinx/examples/python/tutorials/Shors.ipynb [hunk 3]
```diff
@@ -116,29 +136,29 @@
...
+    "            print(\"Found factors of N={} by chance: {} and {}\".format(N, divisor1, divisor2))\n",
     "            return (divisor1, divisor2)\n",
     "\n",
     "        # 3. Find the order of a mod N (i.e., r, where a^r = 1 (mod N))\n",
     "        if quantum == True:\n",
     "            r = find_order_quantum(a, N)\n",
     "        else:\n",
     "            r = find_order_classical(a, N)\n",
-    "        print(\"The order of a is\", r)\n",
+    "        print(\"The order of a ={} is {}\".format(a,r))\n",
```
Comment (bmhowe23) [inline docs/sphinx/examples/python/tutorials/Shors.ipynb:152]:
```suggestion
    "        print(\"The order of a = {} is {}\".format(a,r))\n",
```

**Review by bmhowe23 — APPROVED** [review]
This LGTM 👍

**Comment by mmvandieren** [comment]
[celebrate]     Monica Van Dieren reacted to your message:
________________________________

**Pull request merged**